<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>900</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    border: none; /* 去掉边框 */
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
                                stop: 0 #1A1A1A,  /* 深黑色 */
                                stop: 1 #0D0D0D);  /* 更深的黑色 */
}

QLineEdit {
    background-color: #333;  /* 输入框背景颜色（深灰色） */
    border: 1px solid #666;  /* 输入框边框颜色（浅灰色） */
    border-radius: 5px;  /* 圆角边框 */
    padding: 5px;  /* 内边距 */
    font-size: 14px;  /* 字体大小 */
    color: #FFFFFF;  /* 字体颜色（白色） */
}

QPushButton {
    background-color: #444;  /* 按钮背景颜色（稍浅的黑色） */
    border: none;  /* 去掉边框 */
    border-radius: 5px;  /* 圆角边框 */
    color: #FFFFFF;  /* 字体颜色（白色） */
    font-size: 14px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}

QPushButton:hover {
    background-color: #555;  /* 悬停时的按钮背景颜色（灰色） */
}

QPushButton:pressed {
    background-color: #333;  /* 按下时的按钮背景颜色（更深的灰色） */
}

QLabel {
    color: #FFFFFF;  /* 标签字体颜色（白色） */
    font-size: 14px;  /* 标签字体大小 */
}
</string>
  </property>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/close1.png</normaloff>:/new/prefix1/close1.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>640</width>
     <height>744</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>160</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="res.qrc">:/new/prefix1/moren.png</pixmap>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>50</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>30</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="lineEditId">
                  <property name="minimumSize">
                   <size>
                    <width>515</width>
                    <height>85</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>515</width>
                    <height>85</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>20</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 20pt &quot;宋体&quot;;</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>输入账号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditPassword">
                  <property name="minimumSize">
                   <size>
                    <width>515</width>
                    <height>85</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>515</width>
                    <height>85</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>20</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 20pt &quot;宋体&quot;;</string>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>输入密码</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>已阅读并同意服务协议和传音符隐私保护指引</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonLogin">
            <property name="minimumSize">
             <size>
              <width>512</width>
              <height>68</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>512</width>
              <height>68</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(60, 131, 206);
font: 14pt &quot;宋体&quot;;</string>
            </property>
            <property name="text">
             <string>登录</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonReg">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>145</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#pushButtonReg {
    background-color: transparent;  /* 设置按钮背景为透明 */
    border: none;  /* 去掉边框 */
    color: rgb(0, 170, 255);  /* 设置字体颜色 */
    font-size: 30px;  /* 字体大小 */
    padding: 10px;  /* 内边距 */
}
</string>
          </property>
          <property name="text">
           <string>注册账号</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ip:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIp">
        <property name="text">
         <string>192.168.51.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort">
        <property name="text">
         <string>6699</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
